### DISCUSSION TITLE
Welcome to arco-design-vue Discussions!

### DISCUSSION THREAD

**Discussion by flsion** [arco-design/arco-design-vue#334] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by asher886** [comment]
table 组件  select 事件参数有bug,  列表开启多选选择后把数据删除，列表重新调用接口，再选择新的数据，此时select事件的参数还会把上次已经删掉的数据返回来

**Reply by hehehai** [reply]
@asher886 你好，这应该是符合 table select 的实现的，当出现筛选及分页等情况，仅table中的数据变化不会影响 selected 的（selected 中的 key 是独立的，并未依赖 table data）。若想要select 伴随数据删除也清空的话，可以在数据删除时，将 selected 清空即可

**Reply by asher886** [reply]
table select事件返回的参数数据，怎么清空？？

**Reply by hehehai** [reply]
参考： https://vue-9zaqvp.stackblitz.io
<img width="1009" alt="image" src="https://user-images.githubusercontent.com/12692552/163175001-6bd1b0d9-b133-4ed2-931e-482cebacd7b5.png">
